src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -34,6 +34,8 @@ public static class LanguageServerConstants
         public static class CodeActions
```

Comment:
@NTaylorMullen, I noticed that `LanguageServerConstants` class is public and not internal. Is it the case because we don't have IVT to other assemblies that need it? If so, why not?

Reply:
Not at all. I think that's just an oversight. Should probably be internal unless i'm forgetting something